src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(operand) {
+      calculator.result += operand;
```

Comment:
```suggestion
      this.result += operand;
```

Reply:
Let's practice working with this

Reply:
add(operand) {
      this.result += operand;

      return calculator;
    }
    Test Suites: 1 failed, 1 total
Tests:       9 failed, 9 passed, 18 total
Snapshots:   0 total
Time:        2.65s
Ran all test suites matching /.\\/i.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! js-task-boilerplate@1.0.0 test:only: `mate-scripts test`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the js-task-boilerplate@1.0.0 test:only script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\GIM\AppData\Roaming\npm-cache\_logs\2023-09-09T09_51_35_279Z-debug.log
npm ERR! Test failed.  See above for more details.